.changeset/new-cheetahs-build.md

```diff
@@ -0,0 +1,8 @@
+---
+'braid-design-system': major
+---
+
+Remove treat support
+
+[Treat](https://seek-oss.github.io/treat/) is deprecated and has been superceded by [Vanilla Extract](https://vanilla-extract.style/).
+`.treat.ts` files can no longer be used to for styling and should be converted to `.css.ts` (vanilla extract) files.
```

Comment:
```suggestion
[Treat](https://seek-oss.github.io/treat/) is deprecated and has been superseded by [vanilla-extract](https://vanilla-extract.style/).
`.treat.ts` files can no longer be used to for styling and should be converted to `.css.ts` (vanilla-extract) files.
```

_vanille-extract_ seems to be the preferred spelling (as per [`migration.md`](https://github.com/seek-oss/braid-design-system/blob/remove-treat-support/docs/treat%20to%20vanilla-extract%20migration.md))

And on _supercede/-sede_: https://writingexplained.org/supercede-vs-supersede-difference